recipes/utide/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.0" %}
+{% set commit = "0c612d557f615fbbb7d513892b6840b2302b65e7" %}
+
+package:
+    name: utide
+    version: {{ version }}.dev0
+
+source:
+    fn: {{ commit }}.tar.gz
+    url: https://github.com/wesleybowman/UTide/archive/{{ commit }}.tar.gz
+    sha256: 1000aa9c552309e8e4f5a20f9ed2e62cd1c6bc3794476f5aacc25c2326d6fe08
+
+build:
+    number: 0
+    skip: True  # [win and np>=111]
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - numpy
+        - scipy 0.16.0  # [win]
+        - scipy  # [not win]
+
+test:
+    imports:
+        - utide
+
+about:
+    home: https://github.com/wesleybowman/UTide
+    license: MIT
+    summary: Python distribution of the MatLab package UTide
+
+extra:
+    recipe-maintainers:
+        - ocefpaf
```

Comment:
Thanks @wesleybowman. At this stage these conda recipes are being reviewed by the community and tests in the CIs. Once merged they will live in its own repository that we call feedstock. Everyone listed as maintainers will have commit rights to the feedstock.